Docs: Update SCEP configuration (#14234)
fleetdm/fleet: docs/Using Fleet/MDM-setup.md

fleetctl get mdm-apple
```

## Changing SCEP

The SCEP certificate/key pair generated by Fleet expires every 10 years. 

It's not recommended to change these unless your SCEP certificate or key was compromised.

This is because Fleet uses the pair to encrypt and decrypt macOS hosts' disk encryption (FileVault) keys. 

> If the pair changes, the disk encryption key for all macOS hosts needs to be reset by the end user. The keys will no longer be viewable on each hosts' respective **Host details** page.

Reviewer: ```suggestion
If the pair changes, the disk encryption key for all macOS hosts needs to be reset by the end user. The keys will no longer be viewable on each host's respective **Host details** page.
```

> If your SCEP certificate was been compromised, here's how to change them and reset your disk encryption keys:

Reviewer: ```suggestion
If your SCEP certificate has been compromised, here's how to change it and reset your disk encryption keys:
```
